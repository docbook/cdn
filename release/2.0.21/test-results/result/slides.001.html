<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style="slides"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for slides.001.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: slides.001.xml</h1><p>No changes.</p><section class="foil titlefoil"><article class="page"><header><div class="header-content"><h1>Example slide title</h1></div></header><div class="body"><div class="shownav"><img src="../../resources/img/slides/prev.gif" alt="[Prev]" /><img src="../../resources/img/slides/next.gif" alt="[Next]" /></div><nav class="clicknav"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><span class="personname"><span class="firstname">Jane</span> <span class="surname">Doe</span></span><br /><span class="orgname">Example Corporation</span><br /><span class="pubdate">14 Sep 2015</span></div></article><footer><div class="content"><span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil"><article class="page"><header class="header"><div class="header-content"><h1>Agenda</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><ul class="toc"><li><a href="#1">Introduction</a></li><li><a href="#4">Second foil group</a></li><li><a href="#7">Q&amp;A</a></li></ul></div></article><footer><div class="content"><span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil titlefoil foilgroup"><article class="page"><header><div class="header-content"><h1>Introduction</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav></div></article><footer><div class="content"><span class="foilnumber">Slide 1</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil"><article class="page"><header><div class="header-content"><h1>Learning objectives</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><p>Learn stuff.</p><div class="itemizedlist"><ul><li><p>First point</p></li><li><p>Second point</p></li><li><p>Third point</p></li></ul></div></div></article><footer><div class="content"><span class="foilnumber">Slide 2</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil"><article class="page"><header><div class="header-content"><h1>Something else</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><div class="itemizedlist"><ul><li><p>First point</p></li><li><p>Second point</p></li><li><p>Third point</p></li></ul></div></div></article><footer><div class="content"><span class="foilnumber">Slide 3</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil titlefoil foilgroup"><article class="page"><header><div class="header-content"><h1>Second foil group</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav></div></article><footer><div class="content"><span class="foilnumber">Slide 4</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil"><article class="page"><header><div class="header-content"><h1>More stuff</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><div class="itemizedlist"><ul><li><p>First point</p></li><li><p>Second point</p></li><li><p>Third point</p></li></ul></div></div></article><footer><div class="content"><span class="foilnumber">Slide 5</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil"><article class="page"><header><div class="header-content"><h1>Even more stuff</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><div class="itemizedlist"><ul><li><p>First point</p></li><li><p>Second point</p></li><li><p>Third point</p></li></ul></div></div></article><footer><div class="content"><span class="foilnumber">Slide 6</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil titlefoil foilgroup"><article class="page"><header><div class="header-content"><h1>Q&amp;A</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></nav><p>The ultimate slide...</p></div></article><footer><div class="content"><span class="foilnumber">Slide 7</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script src="../../resources/js/prism.js"></script><script type="text/javascript" language="javascript" src="../../resources/js/jquery-timers-1.2.js"></script><script type="text/javascript" language="javascript" src="../../resources/js/jquery.ba-hashchange.min.js"></script><script type="text/javascript" language="javascript" src="../../resources/js/slides.js"></script><h2>XML source</h2><pre data-src="../src/slides.001.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>