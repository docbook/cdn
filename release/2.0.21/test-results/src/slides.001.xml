<slides xml:lang="en"
        xmlns="http://docbook.org/ns/docbook"
        xmlns:h="http://www.w3.org/1999/xhtml"
        xmlns:xlink="http://www.w3.org/1999/xlink">
<info>
  <title>Example slide title</title>
  <subtitle>Example subtitle</subtitle>
  <author>
    <personname>
      <firstname>Jane</firstname>
      <surname>Doe</surname>
    </personname>
    <affiliation>
      <jobtitle>Lead Engineer</jobtitle>
      <orgname>Example Corporation</orgname>
    </affiliation>
    <email>jane.doe@example.com</email>
  </author>
  <releaseinfo role="version">Version 1.0</releaseinfo>
  <pubdate>2015-09-14</pubdate>
  <copyright>
    <year>2015</year>
    <holder>Example Corporation</holder>
  </copyright>
  <h:meta name="slides.timer" content="true"/>
  <h:meta name="slides.countdown" content="90:00"/>
  <p:style xmlns:p="http://www.w3.org/ns/xproc">slides</p:style>
</info>

<foilgroup>
<title>Introduction</title>

<foil>
<title>Learning objectives</title>
<para>Learn stuff.</para>

<itemizedlist>
<listitem>
<para>First point</para>
</listitem>
<listitem>
<para>Second point</para>
</listitem>
<listitem>
<para>Third point</para>
</listitem>
</itemizedlist>
</foil>

<foil>
<title>Something else</title>

<itemizedlist>
<listitem>
<para>First point</para>
</listitem>
<listitem>
<para>Second point</para>
</listitem>
<listitem>
<para>Third point</para>
</listitem>
</itemizedlist>
</foil>
</foilgroup>

<foilgroup>
<title>Second foil group</title>

<foil>
<title>More stuff</title>

<itemizedlist>
<listitem>
<para>First point</para>
</listitem>
<listitem>
<para>Second point</para>
</listitem>
<listitem>
<para>Third point</para>
</listitem>
</itemizedlist>
</foil>

<foil>
<title>Even more stuff</title>

<itemizedlist>
<listitem>
<para>First point</para>
</listitem>
<listitem>
<para>Second point</para>
</listitem>
<listitem>
<para>Third point</para>
</listitem>
</itemizedlist>
</foil>
</foilgroup>

<foil>
<title>Q&amp;A</title>
<para condition="web">The ultimate slide...</para>
</foil>

</slides>
