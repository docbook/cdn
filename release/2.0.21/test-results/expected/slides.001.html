<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><meta charset="utf-8" /><meta name="localStorage.key" content="slideno" /><title>Example slide title</title><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><link type="text/css" rel="stylesheet" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" media="screen" /><link type="text/css" rel="stylesheet" href="../../resources/css/slides.css" /><meta name="slides.timer" content="true" /><meta name="slides.countdown" content="90:00" /></head><body><section class="foil titlefoil"><article class="page"><header><div class="header-content"><h1>Example slide title</h1></div></header><div class="body"><div class="shownav"><img src="../../resources/img/slides/prev.gif" alt="[Prev]" /><img src="../../resources/img/slides/next.gif" alt="[Next]" /></div><nav class="clicknav"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><span class="personname"><span class="firstname">Jane</span> <span class="surname">Doe</span></span><br /><span class="orgname">Example Corporation</span><br /><span class="pubdate">14 Sep 2015</span></div></article><footer><div class="content"><span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil"><article class="page"><header class="header"><div class="header-content"><h1>Agenda</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><ul class="toc"><li><a href="#1">Introduction</a></li><li><a href="#4">Second foil group</a></li><li><a href="#7">Q&amp;A</a></li></ul></div></article><footer><div class="content"><span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil titlefoil foilgroup"><article class="page"><header><div class="header-content"><h1>Introduction</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav></div></article><footer><div class="content"><span class="foilnumber">Slide 1</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil"><article class="page"><header><div class="header-content"><h1>Learning objectives</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><p>Learn stuff.</p><div class="itemizedlist"><ul><li><p>First point</p></li><li><p>Second point</p></li><li><p>Third point</p></li></ul></div></div></article><footer><div class="content"><span class="foilnumber">Slide 2</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil"><article class="page"><header><div class="header-content"><h1>Something else</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><div class="itemizedlist"><ul><li><p>First point</p></li><li><p>Second point</p></li><li><p>Third point</p></li></ul></div></div></article><footer><div class="content"><span class="foilnumber">Slide 3</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil titlefoil foilgroup"><article class="page"><header><div class="header-content"><h1>Second foil group</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav></div></article><footer><div class="content"><span class="foilnumber">Slide 4</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil"><article class="page"><header><div class="header-content"><h1>More stuff</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><div class="itemizedlist"><ul><li><p>First point</p></li><li><p>Second point</p></li><li><p>Third point</p></li></ul></div></div></article><footer><div class="content"><span class="foilnumber">Slide 5</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil"><article class="page"><header><div class="header-content"><h1>Even more stuff</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><a href="javascript:clicknav('next')"><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></a></nav><div class="itemizedlist"><ul><li><p>First point</p></li><li><p>Second point</p></li><li><p>Third point</p></li></ul></div></div></article><footer><div class="content"><span class="foilnumber">Slide 6</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><section class="foil titlefoil foilgroup"><article class="page"><header><div class="header-content"><h1>Q&amp;A</h1></div></header><div class="body"><nav class="clicknav"><a href="javascript:clicknav('prev')"><img src="../../resources/img/slides/transparent.gif" alt="[Prev]" /></a><img src="../../resources/img/slides/transparent.gif" alt="[Next]" /></nav><p>The ultimate slide...</p></div></article><footer><div class="content"><span class="foilnumber">Slide 7</span>  <span class="copyright">Copyright © <span class="years">2015</span> <span class="holders"><span class="holder">Example Corporation</span></span></span></div></footer></section><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script src="../../resources/js/prism.js"></script><script type="text/javascript" language="javascript" src="../../resources/js/jquery-timers-1.2.js"></script><script type="text/javascript" language="javascript" src="../../resources/js/jquery.ba-hashchange.min.js"></script><script type="text/javascript" language="javascript" src="../../resources/js/slides.js"></script></body></html>